File: PHP/8php 2/remainder.php
<?php
/*
|--------------------------------------------------------------------------
| Find the Remainder
|--------------------------------------------------------------------------
|
*/

function remainder($a, $b) {
   return min($a,$b) != 0 ? max($a,$b) % min($a,$b) : null;
}


function remainder($a, $b) {
   list($a, $b) = $a > $b ? [$a, $b] : [$b, $a];
   return $b === 0 ? null : $a % $b;
}


function remainder($a, $b) {
   if (!$min = min($a, $b)) return null;
   return max($a, $b) % $min;
}


function remainder($a, $b) {
   $arr =array($a, $b);
   return min($arr) === 0 ? NULL : max($arr) % min($arr);
}


print_r(remainder(17, 5));
print_r(remainder(13, 72));
print_r(remainder(0, -1));
print_r(remainder(0, 1));

File: Python/8py 3/remainder.py
"""
|--------------------------------------------------------------------------
| Find the Remainder
|--------------------------------------------------------------------------
|
"""

def remainder(a,b):
   return max(a,b) % min(a,b) if min(a,b) != 0 else None


def remainder(*a):
   a = sorted(a, reverse=True)
   return a[0] % a[1] if a[1] else None


def remainder(a, b):
   a, b = sorted([a, b])
   try:
      return b % a
   except ZeroDivisionError:
      return None


def remainder(a,b):
   if a < b: 
      a, b = b, a      
   return a % b if b else None


print(remainder(17, 5))
print(remainder(13, 72))
print(remainder(0, -1))
print(remainder(0, 1))